New log file opened at Sat May 09 18:47:19 CEST 2015 by 6864@Julia
Taking next connection to: 49269
New client attempting to connect on port: 49270-49269
  Address = /10.0.0.1:49269
This client on port 49270 has uuid 303e6bb8-2377-424b-b52f-183a4b9cb52a configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-303e6bb8-2377-424b-b52f-183a4b9cb52a
Taking next connection to: 49269
New client attempting to connect on port: 49271-49269
  Address = /10.0.0.1:49269
Handshake exception on socket: 49271
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49269
New client attempting to connect on port: 49272-49269
  Address = /10.0.0.1:49269
This client on port 49272 has uuid fcfa7292-f94c-4343-bc33-e39d7178a558 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-303e6bb8-2377-424b-b52f-183a4b9cb52a, activator-fcfa7292-f94c-4343-bc33-e39d7178a558
Taking next connection to: 49269
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49269
New client attempting to connect on port: 49370-49269
  Address = /10.0.0.1:49269
Handshake exception on socket: 49370
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49269
New client attempting to connect on port: 49371-49269
  Address = /10.0.0.1:49269
This client on port 49371 has uuid 36d55ace-24aa-4441-90ea-0e03702af026 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-fcfa7292-f94c-4343-bc33-e39d7178a558, play-fork-run-36d55ace-24aa-4441-90ea-0e03702af026
Taking next connection to: 49269
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49269
